src/components/Accordion/Accordion.test.js

```diff
@@ -39,7 +39,7 @@ describe('Accordion', () => {
         </ExpansionPanel>
       </Accordion>
     );
-    wrapper.children().forEach(panel => {
+    wrapper.children().forEach((panel) => {
```

Comment:
Could we change this lint rule?
Personal preference: I think skipping parens on single argument arrow functions is nice and clean.

Reply:
Yes, we can. I don't mind.